Library/Homebrew/github_packages.rb

```diff
@@ -239,8 +239,8 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, dry_run:)
       when "darwin"
```

Comment:
May want to change `os_version = tab["built_on"]["os_version"] if tab["built_on"].present?` to `os_version = tab["built_on"]["os_version"].presence if tab["built_on"].present?` above too now I think about it so that a blank string for `os_version` will be changed to `nil`.